Modules/IO/JPEG/test/itkJPEGImageIOCMYKTest.cxx

```diff
@@ -46,31 +46,24 @@ itkJPEGImageIOCMYKTest(int argc, char * argv[])
 
     ITK_TRY_EXPECT_NO_EXCEPTION(reader->Update());
 
-    if (!(io->GetPixelType() == itk::CommonEnums::IOPixel::RGB))
-    {
-      std::cout << "Expected RGB. Test failed" << std::endl;
-      return EXIT_FAILURE;
-    }
+    ITK_TEST_EXPECT_TRUE(!(io->GetPixelType() == itk::CommonEnums::IOPixel::RGB));
```

Comment:
Is is wrong.
ITK_TEST_EXPECT_TRUE(io->GetPixelType() == itk::CommonEnums::IOPixel::RGB);

Reply:
:+1: Thanks for catching this @issakomi. Done in d09c797.